crates/ty_python_semantic/resources/mdtest/async.md

```diff
@@ -79,9 +79,8 @@ async def main():
         task("B"),
     )
 
-    # TODO: these should be `int`
-    reveal_type(a)  # revealed: Unknown
-    reveal_type(b)  # revealed: Unknown
+    reveal_type(a)  # revealed: int
+    reveal_type(b)  # revealed: int
```

Comment:
We now understand the [`_FutureLike`](https://github.com/python/typeshed/blob/3e1b90bb2d772781c2b6f318e0f44ef791cf5160/stdlib/asyncio/tasks.pyi#L78-L81) type alias.